hardware/Rtl433.cpp

```diff
@@ -17,9 +17,18 @@
 
 #include "Rtl433.h"
 
-CRtl433::CRtl433(const int ID) :
-	m_stoprequested(false)
+void removeCharsFromString(std::string &str, const char* charsToRemove ) {
```

Comment:
I didn't see it. But if I am not mistaken, stdreplace does only replace one sting at once, so it would need 11 lines of strreplace to do the same thing (one for each char to remove). Tell me what you prefer.